rero-instances/rero-ils-ui/ui-integration.md

```diff
@@ -39,3 +39,166 @@ poetry run folding -c path/to/rero/ng-core -u path/to/rero/rero-ils-ui
 ```
 
 This script must be used after the bootstrap to overwrite it, otherwise the used versions of rero-ils ui and ng-core will be the npm published ones.
+
+## Run only the UI in local
+
+If you need to test a change only in the `rero-ils-ui`, you can use the 
+existing `ils-dev` testing instance as a backend. Make sure the server version 
+is compatible with your local UI.
+
+Temporarily replace `proxy.conf.json` content with:
```

Comment:
This seems dangerous as it can be commit by accident. Can add a warning message or suggest to create a new file and run with a new command `ng serve --proxy-config ...`.